src/main/java/command/DoneCommand.java

```diff
@@ -0,0 +1,54 @@
+package command;
+
+import exception.DoneException;
+import storage.Storage;
+import task.Task;
+import task.TaskList;
+import ui.Ui;
+
+/**
+ * Command for specifying that a task is done.
+ */
+public class DoneCommand extends Command {
+
+    protected String[] input;

```

Comment:
I feel it would be better if you parsed it entirely and then have a class variable of type Task which represents the task that was done?